bin/ycsb

```diff
@@ -78,6 +78,7 @@ DATABASES = {
     "orientdb"     : "com.yahoo.ycsb.db.OrientDBClient",
     "redis"        : "com.yahoo.ycsb.db.RedisClient",
     "s3"           : "com.yahoo.ycsb.db.S3Client",
+    "solr"         : "com.yahoo.ycsb.db.SolrDBClient",
```

Comment:
SolrClient conflicts with the name of an object in the org.apache.solr library that's used as an abstraction through which all communication with a Solr server is routed. I needed to rename it to something.